vignettes/creating-EML.Rmd

```diff
@@ -74,14 +70,21 @@ For this example all of our files are stored in a the Hannon-Example folder with
 "~/data-raw/Hannon-Example".
 
 ```{r}
-excel_path <- "data-raw/Hannon-Example/example-metadata.xlsx"
+excel_path <- system.file("extdata", "Hannon-Example", "example-metadata.xlsx", 
+                          package = "EDIutils", mustWork = TRUE)
+
 sheets <- readxl::excel_sheets(excel_path)
 metadata_sheets <- lapply(sheets, function(x) readxl::read_excel(excel_path, sheet = x))
 names(metadata_sheets) <- sheets
 
-abstract_docx <- "data-raw/Hannon-Example/hannon_example_abstract.docx"
-methods_docx <- "data-raw/Hannon-Example/hannon_example_methods.docx"
-physical_entity <- "data-raw/Hannon-Example/hannon_example_physical_data.csv"
+abstract_docx <- system.file("extdata", "Hannon-Example", "hannon_example_abstract.docx", 
+                             package = "EDIutils", mustWork = TRUE)
+
+methods_docx <- system.file("extdata", "Hannon-Example", "hannon_example_methods.docx", 
+                            package = "EDIutils", mustWork = TRUE)
+
+physical_entity <- system.file("extdata", "Hannon-Example", "hannon_example_physical_data.csv",
```

Comment:
give this a better name, remove example from the title throughout

Reply:
Changed name.